src/components/VCheckbox.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <label :for="id" class="checkbox-label" :class="labelClasses">
+    <input
+      :id="id"
+      type="checkbox"
+      class="checkbox"
+      v-bind="inputAttrs"
+      @change="onChange"
+    />
+    <Checkmark
+      v-show="localCheckedState"
+      class="checkmark"
+      focusable="false"
+      width="20"
+      height="20"
+      role="img"
+    />
+    <!--  @slot The checkbox label  --><slot />
+  </label>
+</template>
+
+<script>
+import Checkmark from '~/assets/icons/checkmark.svg?inline'
+import { computed, defineComponent, ref, watch } from '@nuxtjs/composition-api'
+
+const VCheckbox = defineComponent({
+  name: 'VCheckbox',
```

Comment:
Might be nice to set [the `model` option](https://vuejs.org/v2/api/#model) to the component to use with `v-model`.

Reply:
I wanted to use the `v-model` at first, too. The problem with model for checkbox is it's binding `value` and `input` event. And in checkbox, `value` is the name of the checkbox value (ie., 'CC BY' for licenses), and the thing that actually needs to change is the `checked` property.